TTS/tts/configs/fast_speech_config.py

```diff
@@ -93,8 +93,8 @@ class FastSpeechConfig(BaseTTSConfig):
         binary_loss_alpha (float):
             Weight for the binary loss. If set 0, disables the binary loss. Defaults to 1.0.
 
-        binary_align_loss_start_step (int):
-            Start binary alignment loss after this many steps. Defaults to 20000.
+        binary_loss_warmup_epochs (float):
```

Comment:
"binary_loss_warmup_epochs" is not defined, you renamed the "binary_align_loss_start_step "  to  "binary_align_loss_start_step".  I guess that if you rename  "binary_align_loss_start_step" to "binary_loss_warmup_epochs" and change the default to 150 everything will be ok.

Reply:
Did you check this part?

https://github.com/coqui-ai/TTS/blob/1a5247136290860af5d416dd7c1d73405c5251af/TTS/tts/models/forward_tts.py#L723

Reply:
yes, I mean that "binary_loss_warmup_epochs" is not defined in [TTS/tts/configs/fast_speech_config.py](https://github.com/coqui-ai/TTS/blob/1a5247136290860af5d416dd7c1d73405c5251af/TTS/tts/configs/fast_speech_config.py#L138). You defined "binary_align_loss_start_step: int = 50000" instead of "binary_loss_warmup_epochs: int = 150" in this file (check [this line](https://github.com/coqui-ai/TTS/blob/1a5247136290860af5d416dd7c1d73405c5251af/TTS/tts/configs/fast_speech_config.py#L138))

Reply:
Now should be fixed in ```FastSpeechConfig```. Works for you?